lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/aplinear/APLExplanation.java

```diff
@@ -52,11 +55,17 @@ public double numTotal() {
         return numTotal;
     }
 
+    @JsonProperty("outliers")
+    public double numOutliers() {
+        return numOutliers;
+    }
+
     @Override
     public String prettyPrint() {
         StringBuilder header = new StringBuilder(String.format(
                 "Outlier Explanation:\n"
         ));
+        header.append("Outliers: "+numOutliers+", Total: "+numTotal+"\n");
```

Comment:
This is supposed to print the JSON response, right? It doesn't exactly match the example you put in the commit message—is that okay? (Not sure how strict they are about the output.)

Reply:
no this is the pretty print, the json response is defined by the @Jsonfield decorator

Reply:
Oh, I see—nvm!